tests/Command/CloneSiteCommandTest.php

```diff
@@ -13,64 +13,34 @@
 
 namespace Sonata\PageBundle\Tests\Command;
 
-use PHPUnit\Framework\MockObject\MockObject;
-use PHPUnit\Framework\MockObject\Stub;
 use PHPUnit\Framework\TestCase;
-use Sonata\BlockBundle\Model\BlockManagerInterface;
 use Sonata\PageBundle\Command\CloneSiteCommand;
+use Sonata\PageBundle\Model\BlockManagerInterface;
 use Sonata\PageBundle\Model\PageBlockInterface;
 use Sonata\PageBundle\Model\PageInterface;
 use Sonata\PageBundle\Model\PageManagerInterface;
 use Sonata\PageBundle\Model\SiteInterface;
 use Sonata\PageBundle\Model\SiteManagerInterface;
 use Symfony\Component\Console\Application;
 use Symfony\Component\Console\Tester\CommandTester;
-use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /**
  * @author Christian Gripp <mail@core23.de>
- *
- * NEXT_MAJOR: Remove this legacy group
- *
- * @group legacy
  */
 final class CloneSiteCommandTest extends TestCase
 {
-    /**
-     * @var Application
-     */
-    private $application;
-
-    /**
-     * @var Stub&SiteManagerInterface
```

Comment:
I would revert those, otherwise we miss type information. Same for all test classes.